modules/manage/partials/tiered-storage.adoc

```diff
@@ -1133,6 +1133,76 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true
 
 See also: xref:{topic-recovery-link}[Topic Recovery], xref:manage:kubernetes/k-remote-read-replicas.adoc[Remote Read Replicas]
 
+== Pause and resume uploads
+
+IMPORTANT: Redpanda strongly recommends using pause and resume only under the guidance of Redpanda support or a member of your customer success team. 
```

Comment:
```suggestion
IMPORTANT: Redpanda strongly recommends using pause and resume only under the guidance of https://support.redpanda.com/hc/en-us/requests/new[Redpanda Support^] or a member of your account team. 
```
In other admonitions, we capitalize Redpanda Support or Redpanda Sales & include link. And I've also seen us use "account team" more than "customer success team", but up to you!